docs/faststore/docs/managing-performance/managing-lazy-loading-sections-on-plp.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: "Managing lazy loading sections on Product Listing Pages (PLPs)"
+---
+
+By default, FastStore improves performance by wrapping store sections in `LazyLoadingSection` and ViewportObserver` components. These components delay loading non-essential sections until they are needed, such as when a user scrolls near them. This approach reduces initial page load times and improves user experience, particularly on slower networks or devices.
+
+While lazy loading improves performance, it can harm SEO. Search engine crawlers often do not execute JavaScript or wait for dynamically loaded content. As a result, critical sections like product grids or category pages may not be indexed. For instance, if a user searches for "gym clothes," your product listings might not appear in search results if they are not included in the initial HTML.
+
+In this guide, learn how to optimize SEO by controlling lazy loading for sections in FastStore.
+
+## Before you begin
+
+Make sure the `@faststore/cli` package is updated to the latest version. If you need to update it, follow the instructions on [Updating the '@fastore/cli' package version](https://developers.vtex.com/docs/guides/faststore/project-structure-updating-the-cli-package-version)
+
+## Instructions
+
+### Step 1: Configuring SEO for Product Listing Pages (PLP)
+
+Configure the title and description templates for product listing pages (PLP) in the discovery.config.js file to optimize your store's SEO and allow dynamically generating metadata based on the collection name.
+
+1. Open your FastStore project in a code editor of your preference.
+2. Go to the `discovey.config.js` file and in the `seo` section add the following:
+
+   ```discovery.config.js
+   seo: {
+       plp: {
+       titleTemplate: '%s | FastStore PLP',
+       descriptionTemplate: '%s products on FastStore Product Listing Page',
+       },
+   },
+   ```
+
+   > ⚠ Change the values in `titleTemplate` and `descriptionTemplate` according to your collection’s name.
+
+### Step 2: Controlling lazy loading for custom sections
+
+By default, sections in FastStore are lazy-loaded for performance reasons. However, some sections (e.g., those containing H1 headers) may need to appear in the initial HTML for SEO purposes. To achieve this, you can configure a section to skip lazy loading.
+
+1. Open the `cms/faststore/sections.json` file in your project.
+2. Locate the custom section, add the `skipLazyLoadingSection` property, and set it to `true`. In the following example, we created new custom section called `CustomCollectionName` that displays the store collection name:
```

Comment:
suggestion: we could also link the doc that explains how to create a new custom section
suggestion 2: make it clear that we'll also show an example of the section implementation. I don't know the best way to say this, but something like `First we'll add this section to the sections.json file, and then, on step 3 we'll create this section's component` just to link it better to the next step - but I think step 2 and 3 are so linked that it should be on a single "step section" of the doc 🤔 
```suggestion
2. Locate the custom section, add the `skipLazyLoadingSection` property, and set it to `true`. In the following example, we created a new custom section called `CustomPLPName` that displays the PLP (category/subcategory/collection/brand) name:
```